packages/server/index.js

```diff
@@ -1,16 +1,22 @@
-require('dotenv').config();
-const express = require('express');
-const cors = require('cors');
-const { uploadMiddleware } = require('./middlewares');
-const { transcribeRoute } = require('./routes');
+require("dotenv").config();
+const express = require("express");
+const cors = require("cors");
+const { uploadMiddleware } = require("./middlewares");
+const { transcribeRoute, completionRoute } = require("./routes");
 
 const app = express();
 const PORT = process.env.PORT || 3001;
 
-app.use(express.json({ limit: '50mb' }));
+app.use(express.json({ limit: "50mb" }));
 app.use(cors());
 
-app.post('/transcribe', uploadMiddleware.single('audio'), transcribeRoute);
+app.post("/transcribe", uploadMiddleware.single("audio"), transcribeRoute);
+
+/////
+
+app.get("/completion", (req, res) => {
+  res.status("200").json(completionRoute(req.query.dato))
```

Comment:
the callback should go into the controllers.
Additionally let's change dato per message in the payload, please